openstates/hi/events.py

```diff
@@ -28,70 +22,74 @@ def get_related_bills(self, href):
         get_short_codes(self)
-
+        print("url", URL)
         page = self.lxmlize(URL)
         table = page.xpath(
             "//table[@id='ctl00_ContentPlaceHolderCol1_GridView1']")[0]
 
         for event in table.xpath(".//tr")[1:]:
             tds = event.xpath("./td")
             committee = tds[0].text_content().strip()
-            bills = [x.text_content() for x in tds[1].xpath(".//a")]
             descr = [x.text_content() for x in tds[1].xpath(".//span")]
             if len(descr) != 1:
                 raise Exception
-            descr = descr[0]
+            descr = descr[0].replace('.', '').strip()
             when = tds[2].text_content().strip()
             where = tds[3].text_content().strip()
             notice = tds[4].xpath(".//a")[0]
             notice_href = notice.attrib['href']
             notice_name = notice.text
             when = dt.datetime.strptime(when, "%m/%d/%Y %I:%M %p")
-
-            event = Event(session, when, 'committee:meeting', descr,
-                          location=where)
+            when = pytz.utc.localize(when)
+            event = Event(name=descr, start_time=when, classification='committee-meeting',
+                          description=descr, location_name=where, timezone=tz.zone)
 
             if "/" in committee:
                 committees = committee.split("/")
             else:
-                committees = [committee,]
+                committees = [committee]
 
             for committee in committees:
                 if "INFO" not in committee:
-                    committee = self.short_ids.get("committee",{"chamber":"unknown", "name":committee})
+                    committee = self.short_ids.get("committee", {"chamber": "unknown",
+                                                                 "name": committee})
 
                 else:
                     committee = {
                         "chamber": "joint",
                         "name": committee,
                     }
-
-                event.add_participant('host', committee['name'], 'committee',
-                                      chamber=committee['chamber'])
+                event.add_committee(committee['name'], note='host')
 
             event.add_source(URL)
             event.add_document(notice_name,
                                notice_href,
-                               mimetype='text/html')
-
+                               media_type='text/html')
+            print(notice_name, notice_href)
             for bill in self.get_related_bills(notice_href):
-                event.add_related_bill(
+                a = event.add_agenda_item(description=bill['descr'])
+                a.add_bill(
                     bill['bill_id'],
-                    description=bill['descr'],
-                    type=bill['type']
+                    note=bill['type']
                 )
 
-            self.save_event(event)
+            yield event
+
+    def scrape(self):
```

Comment:
Since this method just calls `scrape_events`, could we drop `scrape_events` and move that code here?